sdk/storage/azure-storage-blob-changefeed/CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
 # Release History
 
-## 12.0.0-beta.9 (Unreleased)
+## 12.0.0-beta.9 (2021-09-15)
+### Dependency updates
 
+- Updated blob dependency to 12.14.0
```

Comment:
same here. Or maybe we just don't ship that one.

Reply:
I think we should continue to release everything in step to make dependency graphs and boms easier

Reply:
for GA packages yes, but I'm not sure if it makes sense to ship any new betas if we didn't add anything.